app/src/Liquidator.ts

```diff
@@ -3,15 +3,18 @@ import { Log } from "web3-core";
 import { Contract } from "web3-eth-contract";
 import { AbiItem } from "web3-utils";
 import LiquidatorABIJson from "./abis/Liquidator.json";
+import MarginAccountABIJson from "./abis/MarginAccount.json";
 import TxManager from "./TxManager";
 import winston from "winston";
 import Bottleneck from "bottleneck";
 
+const ALOE_HEALTHY_ERROR_MESSAGE = "Aloe: healthy";
+const ALOE_GRACE_ERROR_MESSAGE = "Aloe: grace";
 const FACTORY_ADDRESS: string = process.env.FACTORY_ADDRESS!;
 const CREATE_ACCOUNT_TOPIC_ID: string = process.env.CREATE_ACCOUNT_TOPIC_ID!;
 const WALLET_ADDRESS = process.env.WALLET_ADDRESS!;
 const ALOE_INITIAL_DEPLOY = 0;
-const POLLING_INTERVAL_MS = 60_000;
+const POLLING_INTERVAL_MS = 60_000; // 2.5 minutes
```

Comment:
That's not 2.5 minutes

Reply:
good catch, Looks like I forgot to change it